pacman-nodejs-app/deployments/pacman-deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    name: pacman
+  name: pacman
+spec:
+  replicas: 0
+  template:
+    metadata:
+      labels:
+        name: pacman
+    spec:
+      containers:
+      - image: detiber/pacman:latest
```

Comment:
I've temporarily pointed this to the image I've built on dockerhub, since it contains the changes in https://github.com/font/pacman/pull/1, This needs to be updated to point to your image, when possible.